.github/workflows/codeql.yml

```diff
@@ -120,18 +120,3 @@ jobs:
         uses: github/codeql-action/upload-sarif@v3
         with:
           sarif_file: sarif-results/${{ matrix.language }}.sarif
-
-  save_pr_number:
```

Comment:
This is also in `ci.yml`, which calls this workflow script after other prerequisite runs have succeeded.